python-backend/app/commands.py

```diff
@@ -161,3 +163,15 @@ def delete_data():
             db.session.rollback()
             click.echo(f'Error, failed on commit.')
             raise
+
+    if app.config.get('ENVIRONMENT_NAME') == 'prod':
+
+        @sched.app.cli.command()
```

Comment:
Can you explain the use of `@sched` and `sched.` in these changes if this has nothing to do with scheduling? I'm honestly not clear on what these are doing if these changes aren't related to scheduling

Reply:
So, since the function I'm calling lives in the scheduled jobs files I need to use the same application context that it is expecting which is that of sched.app.app_context. So when I register these I need to use the app that is associated with the scheduler.

Reply:
I tried running this using the app.cli.command() instead but since the scheduled jobs or the functions the scheduled jobs are using rely on the context of the app registered to the scheduler it doesn't work. So it has to be registered using the app reference in the scheduler.